spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JdbcEnvironmentRepository.java

```diff
@@ -57,19 +56,21 @@
 
 	private final JdbcTemplate jdbc;
 
-	private final PropertiesResultSetExtractor extractor = new PropertiesResultSetExtractor();
+	private final PropertiesResultSetExtractor extractor;
 
 	private int order;
 
 	private String sql;
 
 	private boolean failOnError;
 
-	public JdbcEnvironmentRepository(JdbcTemplate jdbc, JdbcEnvironmentProperties properties) {
+	public JdbcEnvironmentRepository(JdbcTemplate jdbc, JdbcEnvironmentProperties properties,
```

Comment:
Make this as deprecated and create a new constructor with the addition parameter